script.js

```diff
@@ -0,0 +1,78 @@
+          <li><h3 class="col-lg-12 col-12 col-sm-12">At number four</h3></li>
+          <li><p class="Desc col-lg-12 col-12 col-sm-12">Corpus, Grineer, Infested, Corrupted, you name it, Brauuk has you covered, with his adaptive elemental damage he is ready for everything and everyone.Such a gental soul.</p></li>
+      </ul></li>
+      <li class="Image col-lg-5 col-12 col-sm-12 offset-lg-1 row"> <img class="col-lg-5 col-5 col-sm-5" src="images/mirage.jpg" alt="mirage">
+          <ul class="nav_top col-lg-7 col-7 col-sm-7">
+          <li><h2 class="col-lg-12 col-12 col-sm-12">mirage</h2></li>
+          <li><h3 class="col-lg-12 col-12 col-sm-12">At number five</h3></li>
+          <li><p class="Desc col-lg-12 col-12 col-sm-12">If damage negation and increased damage dealt is not enough...just summon a few clones and your good to go.She has a dico ball thats fun too, maybe not for the enemy but meh...</p></li>
+      </ul></li>
+      <li class="Image col-lg-5 col-12 col-sm-12 row"> <img class="col-lg-5 col-5 col-sm-5" src="images/saryn.jpg" alt="saryn">
+          <ul class="nav_top col-lg-7 col-7 col-sm-7">
+          <li><h2 class="col-lg-12 col-12 col-sm-12">saryn</h2></li>
+          <li><h3 class="col-lg-12 col-12 col-sm-12">At number six</h3></li>
+          <li><p class="Desc col-lg-12 col-12 col-sm-12">Saryn is not someone to be reckoned with. With her spores seeking enemies, making then take extra damage and miasma reducing their armor. Not a gril whos heart you want to break.</p></li>
+      </ul></li>
+  </ul>
+</section>
+<section id="MobileFeaturedSpeakers">
+<h2 class="TopSix col-lg-3 offset-lg-4 col-8 col-sm-8 offset-sm-3 offset-3">Top six used warframes</h2>
+<div class="col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"><hr class="Line col-lg-2 offset-lg-5 col-2 col-sm-2 offset-sm-5 offset-5"></div>
+<ul class="nav_top row col-lg-12 col-12 col-sm-12">
+    <li class="Image col-lg-5 col-12 col-sm-12 offset-lg-1 row"> <img class="col-lg-5 col-5 col-sm-5" src="images/wukong.jpg" alt="wukong">
+        <ul class="nav_top col-lg-6 col-6 col-sm-7">
+        <li><h2 class="col-lg-12 col-12 col-sm-12">Wukong</h2></li>
+        <li><h3 class="col-lg-12 col-12 col-sm-12">Comming in at number one</h3></li>
+        <li><p class="Desc col-lg-12 col-12 col-sm-12">With wukongs celestial twin, he is able to cover both melee and ranged attacks and his cloud walk makes him unstopable when it comes to spy missions.</p></li>
+    </ul></li>
+    <li class="Image col-lg-5 col-12 col-sm-12 row"> <img class="col-lg-5 col-5 col-sm-5" src="images/volt.jpg" alt="volt">
+        <ul class="nav_top col-lg-7 col-7 col-sm-7">
+        <li><h2 class="col-lg-12 col-12 col-sm-12">Volt</h2></li>
+        <li><h3 class="col-lg-12 col-12 col-sm-12">At number two</h3></li>
+        <li><p class="Desc col-lg-12 col-12 col-sm-12">Speed, damage, nuking and tanking...what more do I need to say.</p></li>
+    </ul></li>
+</ul>
+</section>
+`;
```

Comment:
- [ ] The list of speakers used to generate the HTML is assigned to a variable as an array or object 🙃 😉.
```
// You can use an array to store the objects
const speakerList = [ { name: "Speaker Name",... } ]
// It's ok to use an object too
const speakerList = { speaker1: { name: "Speaker Name",... } }
```
- [ ] The object contains all the information needed to generate the dynamic element, at least the name, role, description, and image url about the speaker.
For example:

```javascript
{
  name: "Julia Leda", 
  role: "President of Young pirates of Europe", 
  description: "European integration, political democracy, and participation of ...", 
  photo: "image_url_here"
}
```